src/bundle/Resources/public/js/scripts/button.state.checkbox.toggle.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc) {
+    const toggleForms = [...doc.querySelectorAll('.ez-toggle-btn-state-checkbox')];
+    const ALL_CHECKED = 'all-checked';
+    const ANY_CHECKED = 'any-checked';
+
+    toggleForms.forEach((toggleForm) => {
+        const checkboxInputs = [...toggleForm.querySelectorAll('input[type="checkbox"]')];
+        const toggleButtonState = () => {
+            const button = doc.querySelector(toggleForm.dataset.toggleButtonId);
+
+            const anyIsChecked = checkboxInputs.some(el => el.checked);
```

Comment:
CS: all variables should come one after another without additional empty lines between them.